unlock-app/src/services/storageService.ts

```diff
@@ -47,23 +47,72 @@ interface GetSiweMessageProps {
   chainId: number
   version?: string
 }
+
+interface LoginPromptProps {
+  address: string
+  chainId: number
+  walletService: WalletService
+}
 export class StorageService extends EventEmitter {
   public host: string
 
   public locksmith: LocksmithService
 
+  private accessToken: string | null
+
   constructor(host: string) {
     super()
     this.host = host
     this.locksmith = new LocksmithService({
       host,
     })
+    this.accessToken = null
   }
 
   async login(message: string, signature: string) {
     return this.locksmith.login(message, signature)
   }
 
+  private setToken(token: string) {
```

Comment:
Remove the private. if the function can be used externally, just keep it as it is.